backend/Tournament/views.py

```diff
@@ -51,34 +54,34 @@ def get(self, request, tournament_id):
 		return Response(serializer.data, status=status.HTTP_200_OK)
 
 	@method_decorator(must_be_authenticated)
-	@method_decorator(validate_tournament_request([TournamentState.LOBBY, TournamentState.IN_PROGRESS]))
+	@method_decorator(validate_tournament_request([TournamentState.LOBBY]))
 	def post(self, request, tournament_id):
-		'''If tournament is in lobby, start tournament.
-
-		If tournament is in progress, return next match url.
+		'''starts the tournament.
```

Comment:
Either use docstrings to describe the function properly, or leave it out. We have established the logic already (and if it follows REST, it's almost self-explanatory). If you want to leave me hints, instead of hidden in the code, a sequence diagramme or proper documentation would be appreciated (postman, can also help with that). 

TL;DR no docstring needed here :D